packages/vuetify/src/components/VList/VListGroup.ts

```diff
@@ -1,25 +1,43 @@
 // Components
-import VIcon from '../../components/VIcon'
+import VIcon from '../VIcon'
+import VList from './VList'
 
 // Mixins
 import Bootable from '../../mixins/bootable'
 import Toggleable from '../../mixins/toggleable'
-import {
-  inject as RegistrableInject
-} from '../../mixins/registrable'
+import { Registrable, inject as RegistrableInject } from '../../mixins/registrable'
 
 // Transitions
 import { VExpandTransition } from '../transitions'
 
-/* @vue/component */
-export default {
-  name: 'v-list-group',
+// Utils
+import mixins, { ExtractVue } from '../../util/mixins'
+
+// Types
+import Vue, { VNode } from 'vue'
 
-  mixins: [
-    Bootable,
-    RegistrableInject('list', 'v-list-group', 'v-list'),
-    Toggleable
-  ],
+type VListInstance = InstanceType<typeof VList>
+
+interface options extends Vue {
+  list: VListInstance
+  $route: any
```

Comment:
`	$route: Route`?

Reply:
hmmm. i would assume i would need to import routable (throws error when adding this type)? when adding routable, apparently it doesn't have `path` prop. I thought $route comes from vue router?